app/models/version.rb

```diff
@@ -16,11 +15,11 @@ class Version < ApplicationRecord
   serialize :licenses
   serialize :requirements
 
-  validates :number,   format: { with: /\A#{Gem::Version::VERSION_PATTERN}\z/ }
-  validates :platform, format: { with: Rubygem::NAME_PATTERN }
+  validates :number, length: { maximum: Gemcutter::MAX_FIELD_LENGTH }, format: { with: /\A#{Gem::Version::VERSION_PATTERN}\z/ }
+  validates :platform, length: { maximum: Gemcutter::MAX_FIELD_LENGTH }, format: { with: Rubygem::NAME_PATTERN }
   validates :full_name, presence: true, uniqueness: { case_sensitive: false }
   validates :rubygem, presence: true
-  validates :required_rubygems_version, length: { minimum: 1, maximum: MAX_FIELD_LENGTH }, allow_blank: true
```

Comment:
Let me know if `minimum: 1` was covering a case I am missing. As far as I can tell, this is not needed.

Reply:
I presume it was trying to say that the version is present and not blank.

Reply:
`allow_blank: true` is contradictory with minimum length 1.